### PR TITLE
display: Add missing nullptr check to sceDisplaySetFrameBuf

### PR DESCRIPTION
**Pull request by EXtremeExploit**

### CODE REVIEW

**Comment by EXtremeExploit** [comment]
im not sure if sceDisplayGetFrameBuf needs the same thing

**Comment by Macdu** [comment]
I won't hurt to add the same check to sceDisplayGetFrameBuf

**Review by bookmist — DISMISSED** [review]

File: vita3k/modules/SceDriverUser/SceDisplayUser.cpp [hunk 1]
```diff
@@ -23,6 +23,8 @@ TRACY_MODULE_NAME(SceDisplayUser);
 
 EXPORT(SceInt32, sceDisplayGetFrameBuf, SceDisplayFrameBuf *pFrameBuf, SceDisplaySetBufSync sync) {
     TRACY_FUNC(sceDisplayGetFrameBuf, pFrameBuf, sync);
+    if (!pFrameBuf)
+        return RET_ERROR(SCE_DISPLAY_ERROR_INVALID_VALUE); // is this even correct??, its better than crashing ig
```
Comment (bookmist) [inline vita3k/modules/SceDriverUser/SceDisplayUser.cpp:27]: from disasm ksceDisplayGetFrameBuf (before there is a lot of bypassing values as is)
```  if (pFrameBuf== (SceDisplayFrameBuf *)0x0) {
    return SCE_DISPLAY_ERROR_INVALID_ADDR;
  }
  if ((pFrameBuf->size & 0xfffffffb) != 0x18) {
    return SCE_DISPLAY_ERROR_INVALID_VALUE;
  }```

Comment (EXtremeExploit) [reply vita3k/modules/SceDriverUser/SceDisplayUser.cpp]: should be on par now :D

**Review by bookmist — APPROVED** [review]

**Review by Macdu — APPROVED** [review]

**Pull request merged**